### PR TITLE
Use inputStream when fetching helper messages

### PR DESCRIPTION
**Pull request by SPGoding**
#### Checklist
- [ ] Included tests
- [ ] Tested in MCTEST-xxx

### CODE REVIEW

File: src/main/kotlin/io/github/mojira/arisa/infrastructure/HelperMessages.kt [hunk 1]
```diff
@@ -115,7 +115,9 @@ data class HelperMessages(
         fun deserialize(json: String) = Klaxon().parse<HelperMessages>(json)
 
         fun fetch() = with(URL(url).openConnection() as URLConnection) {
-            deserialize(content as String).rightIfNotNull { Error("Couldn't download or deserialize helper messages") }
+            Klaxon().parse<HelperMessages>(inputStream).rightIfNotNull {
```
Comment (NeunEinser) [inline src/main/kotlin/io/github/mojira/arisa/infrastructure/HelperMessages.kt:118]: Where does the variable `inputStream` come from?

Comment (urielsalis) [reply src/main/kotlin/io/github/mojira/arisa/infrastructure/HelperMessages.kt:118]: he is inside a with, so from the UrlConnection

Comment (SPGoding) [reply src/main/kotlin/io/github/mojira/arisa/infrastructure/HelperMessages.kt:118]: It's a property of `URLConnection`

Comment (NeunEinser) [reply src/main/kotlin/io/github/mojira/arisa/infrastructure/HelperMessages.kt:118]: Oh, the with, it's probably a field. merging.

**Pull request merged**